src/pages/AddItemView.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import db from '../lib/firebase';
+import { collection, addDoc } from 'firebase/firestore';
+import AddItemForm from '../components/AddItemForm';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+export const AddItemView = ({ token }) => {
+  const [inputs, setInputs] = useState({
+    days: '7',
+    'last purchase date': null,
+  });
+
+  console.log(inputs);
+  console.log(token);
+
+  const notify = () => toast('Item added!');
+
+  const handleChange = (e) => {
+    const name = e.target.name;
+    const value = e.target.value;
+
+    setInputs((prevState) => ({ ...prevState, [name]: value }));
+  };
+
+  const handleSubmit = async (e) => {
+    try {
+      e.preventDefault();
+      const docRef = await addDoc(collection(db, token), {
+        item: inputs.item,
+        days: inputs.days,
+        purchased_date: inputs['last purchase date'],
```

Comment:
In the future, it may be good to have strings like these defined as constants that can be used both here and where the name is set in the component. That way you and others don't have to worry about mistyping them.